docs/guide/homeassistant.md

```diff
@@ -208,3 +208,105 @@ The main template is like the thermostat template. The things to add are:
   }
 }
 ```
+# Adding Devices Manually
+
+Some people prefer to not use discovery and so here are some examples of devices you can use:
+
+Note the settings used here for the Gateway in the UI are:
+Type: ValueID topics
+Payload type: Just Value
+Use node name instead of numeric nodeIDs: true
+Ignore location: true
```

Comment:
```suggestion

- **Type**: ValueID topics
- **Payload type**: Just Value
- **Use node name instead of numeric nodeIDs**: true
- **Ignore location**: true
```